x-pack/plugins/cases/public/components/case_view/case_view_tabs.tsx

```diff
@@ -46,7 +46,42 @@ const FilesTab = ({
   </>
 );
 
+const AlertsTab = ({
+  activeTab,
+  totalAlerts,
+  isExperimental,
+}: {
+  activeTab: string;
+  totalAlerts: number | undefined;
+  isExperimental: boolean;
+}) => (
+  <>
+    {ALERTS_TAB}
+    {totalAlerts
+      ? totalAlerts > 0 && (
```

Comment:
Like with the files tab, we should show the `0` if there are no alerts.

Reply:
Hello @cnasikas, there was this comment https://github.com/elastic/kibana/pull/157983#discussion_r1196516266

Reply:
Thanks @gitstart! Now that we get the `totalAlerts` from the `caseData` it will always be defined. I think we should show `0` to be consistent with how the files tab works. Feel free to use the `totalAlerts` variable without any checks.